src/app/lecture/knowledge-node/notes/markdown-editor/markdown-editor.component.html

```diff
@@ -0,0 +1,33 @@
+<div fxLayout="column" class="editor-container">
+  <div *ngIf="mode === 'edit'">
+    <div fxLayout="row" fxLayoutAlign="space-between none">
+      <div fxLayout="row" fxLayoutGap="5px">
+        <mat-icon (click)="insertElement('bold')">format_bold</mat-icon>
+        <mat-icon (click)="insertElement('italic')">format_italic</mat-icon>
+        <mat-divider vertical="true"></mat-divider>
+        <mat-icon [matMenuTriggerFor]="menu">title</mat-icon>
+        <mat-menu #menu class="mat-menu" fxLayout="column">
+          <div (click)="insertElement('h' + h)" *ngFor="let h of [1, 2, 3, 4, 5, 6]" mat-menu-item>
```

Comment:
When we reduce it to h1, h2 and maybe h3 can we change the dropdown to a series of buttons?